app-modules/segment/src/Actions/BulkSegmentAction.php

```diff
@@ -0,0 +1,92 @@
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+namespace AdvisingApp\Segment\Actions;
+
+use Filament\Forms\Form;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\DB;
+use AdvisingApp\Segment\Models\Segment;
+use Filament\Forms\Components\Textarea;
+use Filament\Tables\Actions\BulkAction;
+use Filament\Forms\Components\TextInput;
+use Filament\Notifications\Notification;
+use AdvisingApp\Segment\Enums\SegmentType;
+use AdvisingApp\Segment\Enums\SegmentModel;
+use AdvisingApp\Segment\Jobs\BulkSegmentActionJob;
+
+class BulkSegmentAction
+{
+    public static function make(string $context)
+    {
+        return BulkAction::make('segment')
+            ->icon('heroicon-o-chat-bubble-bottom-center-text')
+            ->label('Create Segment')
+            ->form(fn (Form $form) => $form->schema([
+                TextInput::make('name')
+                    ->label('Name')
+                    ->required()
+                    ->maxLength(255),
+
+                Textarea::make('description')
+                    ->label('Description')
+                    ->maxLength(500),
+            ]))
+            ->action(function (Collection $records, array $data) use ($context) {
+                try {
+                    $data['type'] = SegmentType::Static;
+                    $data['filters'] = [];
+                    $data['model'] = $context == 'students' ? SegmentModel::Student : SegmentModel::Prospect;
+                    $segment = Segment::create($data);
+                    $user = auth()->user();
+                    BulkSegmentActionJob::dispatch($records, $data, $context, $user, $segment);
+                    DB::commit();
+                    Notification::make()
+                        ->title('Segment created')
+                        ->body('The segment has been successfully created and is being processed.')
+                        ->success()
+                        ->send();
+                } catch (\Exception $e) {
```

Comment:
Please import `Exception` into the `use` statements and get rid of the back slash here.